plugins/grib_pi/src/GribUIDialog.h

```diff
@@ -126,6 +147,30 @@ class GRIBUICtrlBar : public GRIBUICtrlBarBase {
   void SetFactoryOptions();
 
   wxDateTime TimelineTime();
+  /**
+   * Retrieves or creates a temporally interpolated GRIB record set for a
+   * specific timestamp.
+   *
+   * This function performs temporal interpolation between two known timestamps
```

Comment:
@sebastien-rosset : I'm no native speaker. That said, I don't think the terms "temporal" and "temporally" are correct in this context. Google says the same. If I got it right it should be "temporary" and "temporarily"

Reply:
I meant "temporal":

> relating to time.

There are at least two types of interpolation:
1. spacial interpolation: given a coordinate, what is the wind speed at that point?
2. temporal interpolation: given a timestamp such as 1AM, what is the wind speed when the forecast has predications for 12AM and 3AM?

I'll read the code more and document.